homework.md

```diff
@@ -51,30 +50,32 @@ A programming language used on the internet
 What three problem does Git & Github solve?
 
 When people want to show off code to each other they can put it on Github
+Contribute to someones else project
+Reverse back the project
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it to your account
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It copy it to my computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you ask to pull changes
```

Comment:
Hi Iwona, just checked your answers, good job with finishing it! If you could look at this question again, and explain a bit more, it would be great help to anyone who is struggling with the Github concepts (also explaining is really hard, but helps you learn more)

Reply:
Thanks, you are absolutely right, will fix that!